exercises/darts/canonical-data.json

```
@@ -70,30 +52,66 @@
     },
     {
       "property": "score",
-      "description": "A dart whose coordinates sum to > 1 but whose radius to origin is <= 1 is scored in the inner circle",
+      "description": "Just within inner circle",
       "input": {
-        "x": 0.4,
-        "y": 0.8
+        "x": 0.7,
+        "y": 0.7
       },
       "expected": 10
     },
     {
       "property": "score",
-      "description": "A dart whose coordinates sum to > 5 but whose radius to origin is <= 5 is scored in the middle circle",
+      "description": "Just beyond inner circle",
       "input": {
-        "x": 2,
-        "y": 4
+        "x": 0.8,
+        "y": -0.8
       },
       "expected": 5
     },
     {
       "property": "score",
-      "description": "A dart whose coordinates sum to > 10 but whose radius to origin is <= 10 is scored in the outer circle",
+      "description": "Just within middle circle",
+      "input": {
+        "x": -3.5,
+        "y": 3.5
+      },
+      "expected": 5
+    },   
+    {
+      "property": "score",
+      "description": "Just beyond middle circle",
```

```suggestion
      "description": "Just outside the middle circle",
```

```suggestion
      "description": "Just within the middle circle",
```

Perhaps "On the outer edge of the middle circle"?